R/wb_load.R

```diff
@@ -196,6 +196,17 @@ wb_load <- function(file, xlsxFile = NULL, sheet, data_only = FALSE) {
     # escape
     workbook_xml <- read_xml(workbookXML, escapes = TRUE)
 
+    if (length(xml_node_name(workbook_xml, stringi::stri_join("x:workbook")))) {
+      op <- options("openxlsx2.has_x_namespace" = TRUE)
+      on.exit(options(op), add = TRUE)
+      out <- paste0(
+        "File has x namespace. We correct this for you, but be careful.\n",
+        "This is somewhat experimental.."
+      )
+      message(out)
```

Comment:
Should this be a `message` or a `warning`?  

Maybe a bit more information on how the x namespace is correct?  I'm not quite sure about what this is, and it could be a little confusing to see this message

Reply:
Tried to make this a bit more user friendly. Wasn't sure if warning or message, but warning might be a good idea. This way it will be easier to spot and we can always decrease this later